index.html

```diff
@@ -640,6 +640,16 @@ <h3>External Binary Content</h3>
           the request entity MUST reject request if it cannot guarantee all of the response headers required by the
           <a>LDP-NR</a> interaction model in this specification.
         </p>
+        <blockquote id="external-content-response-headers-example" class="informative">
+          Non-normative note: Example headers for a successful response to a <code>HEAD</code> request on external
+          content with <code>handling</code> of <code>redirect</code>:
+          <pre>
+Link: &lt;http://www.w3.org/ns/ldp#Resource&gt;;rel="type"
+Link: &lt;http://www.w3.org/ns/ldp#NonRDFSource&gt;;rel="type"
+Link: &lt;http://host/fcrepo/image/fcr:metadata&gt;; rel="describedby"
```

Comment:
Should we show these as separate `Link` headers or combine into one with commas? I think if we show separately we should add a note that they might be combined (per https://tools.ietf.org/html/rfc8288#section-3)